pkg/solana/logpoller/observability.go

```diff
@@ -75,23 +28,23 @@ var _ ORM = &ObservedORM{}
 func NewObservedORM(chainID string, ds sqlutil.DataSource, lggr logger.Logger) *ObservedORM {
 	return &ObservedORM{
 		ORM:           NewORM(chainID, ds, lggr),
-		queryDuration: lpQueryDuration,
-		datasetSize:   lpQueryDataSets,
-		logsInserted:  lpLogsInserted,
+		queryDuration: metrics.LpQueryDuration,
+		datasetSize:   metrics.LpQueryDataSets,
+		logsInserted:  metrics.LpLogsInserted,
```

Comment:
we don't need the `LpBlocksInserted` one defined in `chainlink-framework` repo?

Reply:
Nope, it's not relevant for Solana :)